<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>concentrationv</class>
 <widget class="QWidget" name="concentrationv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConcentrationV</string>
  </property>
  <widget class="QSlider" name="slider_size">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>145</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_size">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>150</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Brush size: 10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>10</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>255</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_concentration">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>150</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Brush concentration: 255</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider_concentration">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>145</width>
     <height>20</height>
    </rect>
   </property>
   <property name="pageStep">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>10</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>255</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="button_clear">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>520</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_load">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>550</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_save">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>550</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_defaults">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>490</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Defaults</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_invert">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>520</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Invert</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>255</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_backgroundconcentration">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Background concentration: 255</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider_backgroundconcentration">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>145</width>
     <height>20</height>
    </rect>
   </property>
   <property name="pageStep">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>10</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_random">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>490</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Random</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
